src/UcrmPluginSdk/Service/UcrmApi.php

```diff
@@ -105,12 +105,12 @@ public static function create(?string $pluginRootPath = null): self
      *
      * @param mixed[] $query
      *
-     * @return mixed[]
+     * @return array|string
```

Comment:
Sorry, the cs check is sometimes too hard on us. :)

```suggestion
     * @return mixed[]|string
```

Reply:
no problem ;)